picky/src/key.rs

```diff
@@ -164,7 +172,7 @@ impl PrivateKey {
 
     pub fn from_pem(pem: &Pem) -> Result<Self, KeyError> {
         match pem.label() {
-            PRIVATE_KEY_PEM_LABEL => Self::from_pkcs8(pem.data()),
+            PRIVATE_KEY_PEM_LABEL | EC_PRIVATE_KEY_LABEL => Self::from_pkcs8(pem.data()),
```

Comment:
This is most likely wrong, for "EC PRIVATE KEY" header value, content is not PKCS8 encoded, it's a "raw" EC key instead. PKCS8 is a container format for various key types such as RSA and EC keys. I'm expecting a separate match branch with `Self::from_ec_der(pem.data())` similar to how it's done for RSA.